code/controllers/subsystem/economy.dm

```diff
@@ -104,3 +112,23 @@ SUBSYSTEM_DEF(economy)
 		return 1
 	inflation_value = max(round(((station_total / bank_accounts_by_id.len) / station_target), 0.1), 1.0)
 	return inflation_value
+
+///
+/**
+ * Proc for adding mail depending on living humans.
+ *
+ * Called by economy fired, though you could call this manually on the SS to add incoming mail with sanity.
+ **/
+/datum/controller/subsystem/economy/proc/generate_mail()
+	var/mail_recipients = list()
+	for(var/mob/living/carbon/human/H in shuffle(GLOB.alive_mob_list))
```

Comment:
1there's way better lists for this that only contain humans, and it shouldn't be shuffled like this either.

Reply:
GLOB.living_crew or something like that